e2e-tests/portal/cypress/e2e/journeys/maker-about-supply-contract/fill-in-about-supply-contract-validation.spec.js

```diff
@@ -36,31 +36,50 @@ context('about-supply-contract', () => {
 
     contractAboutSupplier.visit(deal);
     contractAboutSupplier.supplierName().clear();
+
```

Comment:
The improvements in this test are due to 2x bugs found after updating the GOV design library,  DTFS2-7639, DTFS2-7640.

Reply:
Impressive!